### ISSUE TITLE
Accessibility Bug: The item count information is not conveyed for the list items within the ‘List Mode’.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3149]
Fundamental core component: "Product Switch"

Description: The item count information is not conveyed for the list items within the ‘List Mode’.

Expected: The item count information should be conveyed when the items within the ‘List mode’ is accessed.

Attached screenshot:

![image](https://user-images.githubusercontent.com/32538291/91651637-aca52500-eaac-11ea-956b-fa8343f0344b.png)

**Comment by Lokanathan-k** [comment]
tested with "screen reader extension" found that list count is being read and the count is 12 and the list item found is also 12.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The list count is read.
Hence closing the ticket.
JAWS 2019

Thanks and Regards
Sanjay
